### PR TITLE
feat(api): GET /ops-logs

### PR DESCRIPTION
**Pull request by leeyeh**

### CODE REVIEW

File: docs/api2.yml [hunk 1]
```diff
@@ -672,6 +709,63 @@ paths:
...
+            type: string
+            enum:
+              - createdAt
+              - createdAt-desc
+          description: 排序字段，默认为升序，`-desc` 后缀表示降序
+        - name: pageSize
+          in: query
+          schema:
+            type: integer
+          description: 每页数量，范围：[0, 1000]，默认为 100
```
Comment (sdjdd) [inline docs/api2.yml:737]: `pageSize=0` 没啥意义吧

File: docs/api2.yml [hunk 2]
```diff
@@ -672,6 +709,63 @@ paths:
...
+        - name: page
+          in: query
+          schema:
+            type: integer
+          description: 页数，范围：[1, Infinity)
+        - name: cursor
+          in: query
+          schema:
+            type: string
```
Comment (sdjdd) [inline docs/api2.yml:746]: `format: date-time` :wise-me:

**Review by sdjdd — APPROVED** [review]

File: docs/api2.yml [hunk 3]
```diff
@@ -628,7 +628,7 @@ paths:
         - name: cursor
           in: query
           schema:
-            type: string
+            type: date-time
```
Comment (sdjdd) [inline docs/api2.yml:631]:
```
type: string
fomat: date-time
```
:ganga:

Comment (leeyeh) [reply docs/api2.yml:631]: bfa9dda9f20ecc81c3fbefde655d59110c7b2f6e

**Pull request merged**